Arrays

kathHenderson opened the pull request.

kathHenderson committed:
Adding arrays

kathHenderson committed:
Added control flow stuff

keithbarrow commented on arrays/csvarray.js:
```diff
@@ -0,0 +1,4 @@
+'use strict';
+var hello = "Hello is it me you're looking for?";
```
Good test text 10/10 Would rich-tea again.

keithbarrow commented on arrays/bythree.js:
```diff
@@ -0,0 +1,15 @@
+    ;
+    for (var i = 0; i < theArray.length; i++) {
+        if (theArray[i] % 3 === 0) {
+            console.log(theArray[i]);
```
Just output an array of values divisible by 3.

kathHenderson committed:
Update bythree.js

Changed the function so that the numbers in the array are only multiples of 3 (rather than having two arrays)

keithbarrow commented:
:shipit:
